cmd/merritt/run.go

```diff
@@ -23,6 +22,11 @@ var runCommand = cli.Command{
 			Usage: "path to ui media directory",
 			Value: "ui",
 		},
+		cli.StringFlag{
+			Name:  "docker-url, d",
+			Usage: "docker swarm URL",
+			Value: "http://127.0.0.1:2375",
```

Comment:
It would be nice to support the unix socket out of the box and default to that.  We could do that in a follow up.